addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18502,7 +18502,188 @@ msgctxt "#36552"
 msgid "Defines the number of presentation buffers used by the graphics driver. Select 2 if the driver uses double buffering or 3 for triple buffering."
 msgstr ""
 
-#empty strings from id 36553 to 36597
+#: system/settings/settings.xml
+msgctxt "#36560"
+msgid "Color management"
```

Comment:
In `en_gb`, shouldn't this should be `Colour management`? Same for all other uses of `color/colors` => `colour/colours`.